.github/workflows/pr-tests.yml

```diff
@@ -146,8 +146,10 @@ jobs:
         run: rustup target add riscv32imac-unknown-none-elf --toolchain nightly-2024-08-01-x86_64-unknown-linux-gnu
       - name: Install test dependencies
         run: sudo apt-get update && sudo apt-get install -y binutils-riscv64-unknown-elf lld
-      - name: Run examples
+      - name: Run examples with tagged powdr lib
```

Comment:
wdym tagged powdr lib?

Reply:
the examples that use a tag version of powdr, like 0.1.2. Others use the "local" one in whatever branch it is run

Reply:
maybe change to "with latest powdr release"?